packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,72 @@
+import { theme } from "../styles";
+import { recipe, style } from "../vanilla-extract";
+
+export const wrapper = recipe({
+  base: [
+    {
+      border: "1",
+      fontSize: "md",
+      gap: "xs",
+      overflow: "auto",
+      p: "xs",
+      rounded: "sm",
+      w: "340",
+    },
+    style({
+      letterSpacing: "-0.14px",
+      selectors: {
+        "&:data-invalid": {
+          borderColor: "border.error",
```

Comment:
Its already in 41st line. No need of this anymore.

Reply:
missed it